frappe/core/doctype/system_settings/system_settings.py

```diff
@@ -47,6 +47,13 @@ def on_update(self):
 		frappe.cache().delete_value('time_zone')
 		frappe.local.system_settings = {}
 
+		if self.force_user_to_reset_password:
```

Comment:
on every save? also move to another function

Reply:
Fixed